tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG="$1"
+TO_TAG="$2"
+
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version instead the old one.
+#
+# It will usually be run on a release branch and the resulting changes will be the content of the
+# commit for the basis of the release.
+
+# The script needs two version tags:
+
+# FROM_TAG: the version tag of the previous/current release (e. g. v1.16.7)
+# TO_TAG: the version tag of the new release being prepared (e. g. v1.16.8)
+
+# The two tags should be passed to the script as the first two positional parameters in its invocation.
+
+# So an example invocation could be:
+# set-release-ver.sh v1.16.7 v1.16.8
+#
+# The script replaces all occurrences of $FROM_TAG with $TO_TAG in
+# the relevant affected files.
+
+
+
+
+FILES="
+Documentation/Getting-Started/quickstart.md
+Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md
+Documentation/Upgrade/rook-upgrade.md
+deploy/charts/rook-ceph/values.yaml
+deploy/examples/direct-mount.yaml
+deploy/examples/images.txt
+deploy/examples/operator-openshift.yaml
+deploy/examples/operator.yaml
+deploy/examples/osd-purge.yaml
+deploy/examples/toolbox-job.yaml
+deploy/examples/toolbox-operator-image.yaml
+"
+
+sed -i -e "s/$FROM_TAG/$TO_TAG/g" $FILES
```

Comment:
Using an array resolves the execution error I get on macos.

```suggestion
FILES=(
Documentation/Getting-Started/quickstart.md
Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md
Documentation/Upgrade/rook-upgrade.md
deploy/charts/rook-ceph/values.yaml
deploy/examples/direct-mount.yaml
deploy/examples/images.txt
deploy/examples/operator-openshift.yaml
deploy/examples/operator.yaml
deploy/examples/osd-purge.yaml
deploy/examples/toolbox-job.yaml
deploy/examples/toolbox-operator-image.yaml
)

sed -i -e "s/$FROM_TAG/$TO_TAG/g" "${FILES[@]}"
```